packages/beacon-node/src/network/reqresp/beaconBlocksMaybeBlobsByRoot.ts

```diff
@@ -575,71 +574,44 @@ export async function unavailableBeaconBlobsByRootPostFulu(
     return acc;
   }, [] as number[]);
 
-  let resolveBlockInput: ((block: BlockInput) => void) | null = null;
-  const blockInputPromise = new Promise<BlockInput>((resolveCB) => {
-    resolveBlockInput = resolveCB;
-  });
-  if (resolveBlockInput === null) {
-    throw Error("Promise Constructor was not executed immediately");
-  }
-
-  const gotColumnsFromExecution = await getDataColumnsFromExecution(
```

Comment:
the only change in this file is to remove `getDataColumnsFromExecution` call, but it caused a lot of diffs due to indentation